lib/pages/moedas_detalhes_page.dart

```diff
@@ -136,8 +138,8 @@ class _MoedasDetalhesPageState extends State<MoedasDetalhesPage> {
                   verticalDirection: VerticalDirection.down,
                   children: [
                     ListTile(
-                      title: const Text(
-                        "Moeda",
+                      title: Text(
+                        AppLocalizations.of(context)!.nameCoin,
```

Comment:
Da forma que você declarou pageName no começo da página, você poderia usar isso para tudo, então, eu sugiro mudar o nome da variável para algo mais entendível, como alguma das sugestões de nome que a Júlia deu. Nessa linha que eu marquei por exemplo, você colocaria (nome da variável).nameCoin e não precisaria estar chamando o AppLocalizations.of(context)! todas as vezes